.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 3.7.0
+current_version = 3.7.1
```

Comment:
Hey @sbneto @alanjds @mjbradford89 what you think about the version? should we bump v4.0 instead now that we are breaking compatibility with old versions of Python? specially the removal of Python 2.7, or maybe just bump v2.8 instead.

Reply:
I think it could even be a major version since it breaks compatibility with some previous code, but at least a minor.